README.md

```diff
@@ -46,25 +46,25 @@ To start working on the Agent, you can build the `master` branch:
 2. cd into the project folder: `cd $GOPATH/src/github.com/DataDog/datadog-agent`.
 3. Install project's dependencies: `invoke deps`.
    Make sure that `$GOPATH/bin` is in your `$PATH` otherwise this step might fail.
-4. Build the `rtloader` dependency with `invoke rtloader.build && invoke
-   rtloader.install`. You will need CMake installed and a C++ compiler for this
-   to work.
-   `rtloader` is in charge of loading and running Python. By default `rtloader`
-   will be built for Python2, but you can choose which versions of Python you want
+4. Create a development `datadog.yaml` configuration file in `dev/dist/datadog.yaml`, containing a valid API key: `api_key: <API_KEY>`
+5. Build the agent with `invoke agent.build --build-exclude=systemd`. 
+   By default the agent will be built for Python3, but you can choose which versions of Python you want
```

Comment:
typo: which version
Or, a proposal: `By default, the Agent will be built to use Python 3 but you can select which Python version you want to use`